### PR TITLE
chore:  Added Discord Bot Project

### PR DESCRIPTION
**Pull request by killerraj369**
## Related Issue
Issue No - #1201

## Description
A Discord Bot for Coders. Now Contests and Hackathons related Information under your command. Executes Code in Discord Itself. Provides Jokes, News, Quotes and fun things. Supported - C, C++, Python, Java, JavaScript, Golang, C# etc.

### CODE REVIEW

**Comment by killerraj369** [comment]
Kindly Review the PR, and tell if changes are required else you can merge the PR

File: src/DB/projects.json [hunk 1]
```diff
@@ -2322,5 +2322,23 @@
...
     ]
+  },
+  {
+      "github_username": "killerraj369",  
+      "Social_media": {
+        "gitHub": "https://github.com/killerraj369",
+        "LinkedIn": "https://www.linkedin.com/in/vaibhav-raj-229967218/",
+        "Instagram": "",
+        "YouTube": "",
+        "Twitter": ""
```
Comment (priyankarpal) [inline src/DB/projects.json:2333]: remove these code if you're not using

File: src/DB/projects.json [hunk 2]
```diff
@@ -2322,5 +2322,23 @@
...
+      },
+      "Projects": [
+        {
+          "link": "https://github.com/killerraj369/CodeHelper",
+          "title": "CodeHelper Discord Bot",
+          "description": "A Discord Bot for Coders. Now Contests and Hackathons related Information under your command. Executes Code in Discord Itself. Provides Jokes, News, Quotes and fun things. Supported - C, C++, Python, Java, JavaScript, Golang, C# etc.",
+          "tech": ["Node.js", "Express.js","Discord.js","RapidApi"]
+        },
```
Comment (priyankarpal) [inline src/DB/projects.json:2341]: remove the comma `,`
```suggestion
        }
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by killerraj369** [comment]
Hey @priyankarpal  I have made all the required changes and resolved conflicts.  All Checks have also passed.
Kindly Take a review and merge my PR.

**Review by killerraj369 — COMMENTED** [review]
I have made all the changes required

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**